### PR TITLE
Added Metric for pull request with messages

### PR DESCRIPTION
**Pull request by pratikmishra356**
@ccarterlandis @gabe-heim Created new metrics for pull requests with messages

### CODE REVIEW

File: augur/metrics/pull_request/pull_request.py [hunk 1]
```diff
@@ -427,5 +427,37 @@ def pull_request_acceptance_rate(self, repo_group_id, repo_id=None, begin_date=N
                                                         'begin_date': begin_date, 'end_date': end_date})
         return results
 
+@annotate(tag='pull-request-message')
+def pull_request_message(self, repo_group_id, repo_id=None):
```
Comment (mrsaicharan1) [inline augur/metrics/pull_request/pull_request.py]: Please read up the documentation on how to add a new metric. You're missing some keyword arguments.

**Comment by mrsaicharan1** [comment]
@pratikmishra356 Your commit history is messed up. Please do fix it up!

**Comment by mrsaicharan1** [comment]
Hi @pratikmishra356,
 How is this PR any different from https://github.com/chaoss/augur/pull/651?

**Comment by pratikmishra356** [comment]
@mrsaicharan1 I think #651 returns only count for comments, but this pr will give text messages

File: augur/metrics/pull_request/pull_request.py (hunk 1 above)
Comment (mrsaicharan1) [reply augur/metrics/pull_request/pull_request.py]: > @mrsaicharan1 I think #651 returns only count for comments, but this pr will give text messages

Umm, you might've been mistaken. PR #651 returns PRs with comments.

**Comment by mrsaicharan1** [comment]
Each result you return is a group of data points consisting of time and some calculated value. I don't think there's any use for text messages!

**Comment by pratikmishra356** [comment]
@mrsaicharan1 No,Actually for Anomaly detection we also need to analyse comments messages so we need that.

**Comment by mrsaicharan1** [comment]
Screenshot 2020-03-30 at 9 16 10 PM
You're not utilizing a time series & value! Ergo, this does not qualify as a metric!

**Comment by mrsaicharan1** [comment]
If you really need the message info, you can add it separately on top of #651 . This PR would just be a duplicate of it.

**Comment by pratikmishra356** [comment]
Screen Shot 2020-03-30 at 9 19 39 PM
@mrsaicharan1 your pr is combined with issues not will pull requests however we also need messages for issues that can be combined in your PR

**Comment by mrsaicharan1** [comment]
> Screen Shot 2020-03-30 at 9 19 39 PM
> @mrsaicharan1 your pr is combined with issues not will pull requests however we also need messages for issues that can be combined in your PR

Okay! This makes it right!
But you still have a few things left out namely,
1. Metric and API tests are missing. Have you tested this locally?
2. Commit history is messed up.

**Comment by pratikmishra356** [comment]
Screen Shot 2020-03-30 at 9 27 12 PM
Yes I tested it locally and I will add API and metric tests
I am trying to clear up this commit history mess

**Comment by mrsaicharan1** [comment]
Maybe this could help out. If you got any queries, you can post them here :smile:


https://github.com/chaoss/augur/pull/649#issuecomment-605674476

**Comment by pratikmishra356** [comment]
@mrsaicharan1 Thanks, It would be nice working with you , thanks for your help.
I think I had cleared the commit's mess.

**Comment by pratikmishra356** [comment]
Sorry , I had some conflicts with files so I am closing this PR

**Pull request closed without merging**